python/main.py

```diff
@@ -28,28 +30,31 @@ def root():
 def add_item(name: str = Form(...), category: str = Form(...)):
     logger.info(f"Receive item: {name} Category: {category}")
 
-    filename = "items.json"
-    new_item = {"name": name, "category": category}
-    
-    with open(filename, "r") as file:
-        data = json.load(file)
-    
-    data["items"].append(new_item)
-
-    with open(filename, "w") as file:
-        json.dump(data, file)
+    dbname = "mercari.sqlite3"
```

Comment:
dbname can be declared outside the function so every function can access to it.

Reply:
You're right, thanks for the comment!